std/exception.d

```diff
@@ -1191,8 +1194,33 @@ if (__traits(isRef, source) || isDynamicArray!S ||
     }
     else static if (isStaticArray!S)
     {
-        foreach (size_t i; 0 .. S.length)
-            if (mayPointTo(source[i], target)) return true;
+        static if (is(S == void[n], size_t n))
+        {
+            static if (n >= (void*).sizeof)
+            {
+                // Reinterpreting cast is impossible during ctfe
+                if (__ctfe)
+                    return true;
```

Comment:
I tried to make this work, you should be able to cast to some form of `ubyte[]`, but the compiler refuses to see anything but a `void[N]`. I tried slicing, slicing a pointer, and using a helper function.